persistence/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/PersistenceComponents.scala

```diff
@@ -31,9 +31,11 @@ trait ReadSidePersistenceComponents extends WriteSidePersistenceComponents {
   def actorSystem: ActorSystem
   def executionContext: ExecutionContext
   def materializer: Materializer
-  def configuration: Configuration
 
-  lazy val readSideConfig: ReadSideConfig = ReadSideConfig(configuration)
+  def conf: Config = configuration.underlying
```

Comment:
Let's call it `config` for consistency with the type name and the convention elsewhere in the code. This one actually _is_ a public API, so the name is significant.